frappe_slack_connector/db/timesheet.py

```diff
@@ -4,6 +4,59 @@
 from frappe_slack_connector.db.employee import get_employee_from_user
 
 
+def get_user_projects(user: str, limit: int | None = 90) -> list:
+    """
+    Get the projects for the given user
+    NOTE: `set_user()` will effectively wipe out frappe.form_dict
+    """
+    frappe.set_user(user)
```

Comment:
Set this only once, we can add after this line : https://github.com/rtCamp/frappe-slack-connector/pull/52/files#diff-5f024dea7b3a2aa753a359e446137b36decb7a0e9a9cf86efb0e46318eb3cc08R29

Reply:
Moved to the controller